group_fixup.py

```diff
@@ -276,6 +276,22 @@ def provision_group(gid):
     return call_api3(POST, path, data)
 
 
+def provision_group_members(gid):
+    prov_id = options.prov_id
+    def path(pid): return f"co_provisioning_targets/provision/{prov_id}/copersonid:{pid}.json"
```

Comment:
```suggestion
    def path(pid, prov_id): return f"co_provisioning_targets/provision/{prov_id}/copersonid:{pid}.json"
```

For sanity's sake, `prov_id` should probably be a positional arg as well. Though if this is just a function that's only used in one spot that just returns an f-string, it feels like we should just use the f-string at the call site.

Reply:
I agree, it's less strange to define `path` as an f-string right before the API call. I've pushed that change.

(See my other response re: `options` and `prov_id`.)